arches/app/etl_modules/base_import_module.py

```diff
@@ -46,6 +46,9 @@ def __init__(
         if self.request:
             self.userid = request.user.id
             self.moduleid = request.POST.get("module")
+            self.loadid = request.POST.get("load_id")
```

Comment:
Setting the loadid used to be run in `prepare_temp_dir()`, but was removed in 8532cc69b90acbc89fda394ad046d704740dac77.

Reply:
@jacobtylerwalls Sorry about this. This should've been added to the base importer init, when getting the loadid from the request was removed other places. At one point, one is overwritten by the other and that creates the issue.